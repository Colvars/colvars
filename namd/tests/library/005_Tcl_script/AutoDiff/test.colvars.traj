# step         d                     ft_d                  fa_d                    E_harmonic1          
           0    0.00000000000000e+00  0.00000000000000e+00  3.00000000000000e+03   1.80000000000000e+04
# step         d                     ft_d                  fa_d                    E_harmonic1          
           0    1.42467449615693e+01  0.00000000000000e+00 -5.61686240392322e+02   6.30982865292123e+02
           2    1.42282559728026e+01  5.32933601891204e+00 -5.59529927203773e+02   6.20640585041317e+02
           4    1.41727662460744e+01  3.36254334374946e+01 -5.43191561518601e+02   5.90114145010033e+02
           6    1.40831323320990e+01  8.13697354080271e+01 -5.89087882343440e+02   5.42430039129534e+02
           8    1.39609835871970e+01  1.43683712693643e+02 -4.90245896799256e+02   4.80682078657013e+02
          10    1.38136915830229e+01  2.11747525548495e+02 -6.48260269460101e+02   4.11184644791030e+02
          12    1.36437184346326e+01  2.81640059905252e+02 -6.92569668563411e+02   3.37726286543895e+02
          14    1.34588898265340e+01  3.45729335512429e+02 -7.10451792145928e+02   2.66044940745548e+02
          16    1.32645132385777e+01  4.00985013276056e+02 -7.17113322920477e+02   1.99874216317278e+02
          18    1.30658878095430e+01  4.46271310459046e+02 -7.12743262844788e+02   1.42014602816538e+02
          20    1.28684277602171e+01  4.80580457979151e+02 -6.97687398033433e+02   9.42708468394674e+01
# step         d                     ft_d                  fa_d                    E_harmonic1            E_harm2              
          20    1.28684277602171e+01  4.91888943151427e+02 -7.00052334624999e+02   9.42708468394674e+01  2.16390678518517e-01
          22    1.26751474320475e+01  5.03084201909373e+02 -6.63749745722433e+02   5.69780068750397e+01  1.31911488631676e+00
          24    1.24888492973110e+01  5.15335038327460e+02 -6.24768595087998e+02   2.98717044351878e+01  3.26593801074265e+00
          26    1.23139243019631e+01  5.23015189033541e+02 -5.84344372073400e+02   1.23185584203791e+01  5.88374829295988e+00
          28    1.21466432033769e+01  5.30527139707869e+02 -1.53268809286588e+01   2.68802863708136e+00  9.10272277928461e+00
          30    1.19993450537562e+01  5.38695710361251e+02  2.51801102170466e+01   5.36193227844337e-05  1.25163790180274e+01
          32    1.18850444890963e+01  5.34994463383540e+02  5.66127654985307e+01   1.65184618589271e+00  1.55390723911830e+01
          34    1.18115458381165e+01  5.01682267555293e+02  7.68248945179519e+01   4.43937139139973e+00  1.76552911862265e+01
          36    1.17769967622096e+01  4.41508793313709e+02  8.63258903923681e+01   6.21630550812685e+00  1.86967114955734e+01
          38    1.17769542139854e+01  3.61640408006584e+02  8.63375911540067e+01   6.21867783235716e+00  1.86980124336000e+01
          40    1.18061287066284e+01  2.69866689808628e+02  7.83146056771794e+01   4.69825979919525e+00  1.78166083142086e+01
# step         d                     ft_d                  fa_d                    E_harm2              
          40    1.18061287066284e+01  2.21949532217238e+02  2.98467823342890e+01   1.78166083142086e+01
          42    1.18555777458287e+01  1.74464230745251e+02  2.86105563542814e+01   1.63712786980303e+01
          44    1.19176677646512e+01  8.22937404700844e+01  2.70583058837192e+01   1.46430383459383e+01
          46    1.19851813910175e+01 -6.16868563783110e+00  2.53704652245623e+01   1.28732101142145e+01
          48    1.20415309208210e+01 -8.97664352869559e+01 -3.04621864055419e+02   1.14832871967769e+01
          50    1.20655400380907e+01 -1.50412424518311e+02 -1.19070572741895e+02   1.09151927551443e+01
          52    1.20686235841253e+01 -1.88148861003765e+02 -2.70255157752960e+01   1.08432753505955e+01
          54    1.20571466466005e+01 -2.05067811286631e+02  2.93687990112175e+01   1.11121555752091e+01
          56    1.20332538278214e+01 -2.00399869950163e+02  6.58114558084562e+01   1.16824770177739e+01
          58    1.20000609172601e+01 -1.75170570145191e+02  8.83273231404457e+01   1.24984771148846e+01
          60    1.19614725548519e+01 -1.36757725550194e+02  2.59631861287023e+01   1.34817406790728e+01
