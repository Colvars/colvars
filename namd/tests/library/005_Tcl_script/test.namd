# -*- tcl -*-

# FILENAMES
set outName             [file rootname [file tail [info script]]]
#set inFileNum           [expr [scan [string range $outName end-1 end] "%d"] - 1]
#set inName              [format "%s%02u" [string range $outName 0 end-2] $inFileNum]

# FILES

#outputname              test01
#DCDfile                 test01.dcd

# MD SETUP
timestep          0.5
numSteps		  20

# Interactive molecular dynamics
#IMDon               on
#IMDport             3000
#IMDfreq             20
#IMDwait             on

colvars			  on
colvarsConfig	  test.in
#colvarsInput  $inName.colvars.state
#colvarsInput  test.colvars.state

#tclForces on
#tclForcesScript {
#  addatom 12
#  proc calcforces {} {
#    addforce 12 { 10 0 0 }
#  }
#}

# INPUT
structure               ../Common/da.psf 
#parameters              missing.par
parameters              ../Common/par_all22_prot.inp
paraTypeCharmm          on
coordinates             ../Common/da.min.pdb

temperature 250
#bincoordinates       $inName.coor
#binvelocities        $inName.vel

# OUTPUT

outputname              ${outName}
outputenergies          100
outputtiming            2000
binaryoutput            no
# binaryrestart           yes

#XSTFreq                 1000

# DCD TRAJECTORY

DCDfile                 ${outName}.dcd
DCDfreq                 2000

seed		87654321

# CONSTANT-T
langevin                on
langevinTemp		    300.0
langevinDamping         10.0
langevinHydrogen	    on

# SPACE PARTITIONING
splitpatch              hydrogen
hgroupcutoff            2.8
stepspercycle           20
margin                  1.0

# CUT-OFFS
switching               on
switchdist              9.0
cutoff                  11.0
pairlistdist            12.0

# RESPA 
fullElectFrequency      1
nonbondedFreq           1

# 1-4 NON-BONDED
exclude                 scaled1-4
1-4scaling              1.0

# COM
commotion               no

# SHAKE
rigidBonds              none

proc runcmd { cmd } {
  print "Running $cmd"
  print [eval $cmd]
}

foreach c [cv list] { print "$c = [cv colvar $c value]"}        
foreach b [cv list biases] { print "E($b) = [cv bias $b energy]"}

runcmd "cv update"

foreach c [cv list] { print "$c = [cv colvar $c value]"}        
foreach b [cv list biases] { print "E($b) = [cv bias $b energy]"}

run 20

foreach c [cv list] { print "$c = [cv colvar $c value]"}        
foreach b [cv list biases] { print "E($b) = [cv bias $b energy]"}

# cv config "colvar {
#   name d2
#   distance {
#     group1 { atomnumbers 4 }
#     group2 { atomnumbers 99 }
#   }
# }"
# runcmd "cv colvar d2 value"
# runcmd "cv update"
# runcmd "cv colvar d2 value"
# run 20
# runcmd "cv colvar d2 value"
# runcmd "cv colvar d2 delete"

cv config "harmonic {
  name harm2
  colvars d
  centers 13
  forceConstant 1.
  outputEnergy yes
}"
run 20 

runcmd "cv bias harmonic1 state"
runcmd "cv bias harmonic1 delete"

runcmd "cv bias harm2 energy"
runcmd "cv bias harm2 update"
run 20 
runcmd "cv bias harm2 energy"
runcmd "cv bias harm2 state"
runcmd "cv bias harm2 delete"

cv reset
# reinitialize just for the fun of it ;-)
cv configfile 	  test.in
