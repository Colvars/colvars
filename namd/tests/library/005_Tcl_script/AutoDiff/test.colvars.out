colvars: ----------------------------------------------------------------------
colvars: Please cite Fiorin et al, Mol Phys 2013:
colvars:  https://dx.doi.org/10.1080/00268976.2013.813594
colvars: in any publication based on this calculation.
colvars: SMP parallelism is enabled; if needed, use "smp off" to override this.
colvars: This version was built with the C++11 standard or higher.
colvars: ----------------------------------------------------------------------
colvars: Reading new configuration from file "test.in":
colvars: # units = "" [default]
colvars: # smp = on [default]
colvars: # colvarsTrajFrequency = 2
colvars: # colvarsRestartFrequency = 0 [default]
colvars: # scriptedColvarForces = off [default]
colvars: # scriptingAfterBiases = off [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = "d"
colvars:   Initializing a new "distance" component.
colvars:     # name = "" [default]
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # forceNoPBC = off [default]
colvars:     # scalable = on [default]
colvars:       Initializing atom group "group1".
colvars:       # name = "" [default]
colvars:       # centerReference = off [default]
colvars:       # rotateReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableForces = on [default]
colvars:       # enableFitGradients = on [default]
colvars:       Enabling scalable calculation for group "group1".
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "group1" defined with 1 atoms requested: total mass = 12.011, total charge = -0.1.
colvars:       Initializing atom group "group2".
colvars:       # name = "" [default]
colvars:       # centerReference = off [default]
colvars:       # rotateReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableForces = on [default]
colvars:       # enableFitGradients = on [default]
colvars:       Enabling scalable calculation for group "group2".
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "group2" defined with 1 atoms requested: total mass = 12.011, total charge = 0.07.
colvars:     # oneSiteSystemForce = off [default]
colvars:     # oneSiteTotalForce = off [default]
colvars:   All components initialized.
colvars:   # timeStepFactor = 1 [default]
colvars:   # width = 0.2
colvars:   # lowerBoundary = 12
colvars:   # upperBoundary = 32
colvars:   # hardLowerBoundary = off [default]
colvars:   # hardUpperBoundary = off [default]
colvars:   # expandBoundaries = off [default]
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputTotalForce = on
colvars:   # outputAppliedForce = on
colvars:   # subtractAppliedForce = on
colvars:   # runAve = off [default]
colvars:   # corrFunc = off [default]
colvars: ----------------------------------------------------------------------
colvars: Collective variables initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new "abf" instance.
colvars:   # name = "abf1" [default]
colvars:   # colvars = { d }
colvars:   # stepZeroData = off [default]
colvars:   # outputEnergy = off [default]
colvars:   # outputFreq = 60
colvars:   # timeStepFactor = 1 [default]
colvars:   # applyBias = on [default]
colvars:   # updateBias = on [default]
colvars:   # hideJacobian = off [default]
colvars:   Jacobian (geometric) forces will be included in reported free energy gradients.
colvars:   # fullSamples = 0
colvars:   # inputPrefix =  [default]
colvars:   # historyFreq = 0 [default]
colvars:   # shared = off [default]
colvars:   # maxForce =  [default]
colvars:   # integrate = on [default]
colvars:   Finished ABF setup.
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new "harmonic" instance.
colvars:   # name = "harmonic1" [default]
colvars:   # colvars = { d }
colvars:   # stepZeroData = off [default]
colvars:   # outputEnergy = on
colvars:   # outputFreq = 0 [default]
colvars:   # timeStepFactor = 1 [default]
colvars:   # writeTISamples = off [default]
colvars:   # writeTIPMF = off [default]
colvars:   # centers = { 12 }
colvars:   # targetCenters = { 12 } [default]
colvars:   # outputCenters = off [default]
colvars:   # forceConstant = 10
colvars:   # targetForceConstant = -1 [default]
colvars:   The force constant for colvar "d" will be rescaled to 250 according to the specified width (0.2).
colvars: ----------------------------------------------------------------------
colvars: Collective variables biases initialized, 2 in total.
colvars: ----------------------------------------------------------------------
colvars: Collective variables module (re)initialized.
colvars: ----------------------------------------------------------------------
colvars: Updating NAMD interface:
colvars: updating atomic data (0 atoms).
colvars: updating group data (2 scalable groups, 2 atoms in total).
colvars: Re-initialized atom group for variable "d":0/0. 1 atoms: total mass = 12.011, total charge = -0.1.
colvars: Re-initialized atom group for variable "d":0/1. 1 atoms: total mass = 12.011, total charge = 0.07.
colvars: The final output state file will be "test.colvars.state".
colvars: Opening trajectory file "test.colvars.traj".
colvars: Redefining the Tcl "cv" command to the new script interface.
colvars:   Prepared sample and gradient buffers at step 0.
colvars: Updating NAMD interface:
colvars: updating atomic data (0 atoms).
colvars: updating group data (2 scalable groups, 2 atoms in total).
colvars: Re-initialized atom group for variable "d":0/0. 1 atoms: total mass = 12.011, total charge = -0.1.
colvars: Re-initialized atom group for variable "d":0/1. 1 atoms: total mass = 12.011, total charge = 0.07.
colvars: The final output state file will be "test.colvars.state".
colvars: Saving collective variables state to "test.colvars.state".
colvars: ----------------------------------------------------------------------
colvars: Reading new configuration:
colvars: # units = "" [default]
colvars: # smp = on [default]
colvars: # colvarsTrajFrequency = 2 [default]
colvars: # colvarsRestartFrequency = 0 [default]
colvars: # scriptedColvarForces = off [default]
colvars: # scriptingAfterBiases = off [default]
colvars: ----------------------------------------------------------------------
colvars: Collective variables initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new "harmonic" instance.
colvars:   # name = "harm2"
colvars:   # colvars = { d }
colvars:   # stepZeroData = off [default]
colvars:   # outputEnergy = on
colvars:   # outputFreq = 0 [default]
colvars:   # timeStepFactor = 1 [default]
colvars:   # writeTISamples = off [default]
colvars:   # writeTIPMF = off [default]
colvars:   # centers = { 13 }
colvars:   # targetCenters = { 13 } [default]
colvars:   # outputCenters = off [default]
colvars:   # forceConstant = 1
colvars:   # targetForceConstant = -1 [default]
colvars:   The force constant for colvar "d" will be rescaled to 25 according to the specified width (0.2).
colvars: ----------------------------------------------------------------------
colvars: Collective variables biases initialized, 3 in total.
colvars: ----------------------------------------------------------------------
colvars: Collective variables module (re)initialized.
colvars: ----------------------------------------------------------------------
colvars: Updating NAMD interface:
colvars: updating atomic data (0 atoms).
colvars: updating group data (2 scalable groups, 2 atoms in total).
colvars: Saving collective variables state to "test.colvars.state".
colvars: Features of "colvar d" ON/OFF (refcount)
colvars: - active   ON (4)
colvars: - awake   ON (0)
colvars: - gradient   ON (3)
colvars: - collect_gradient   OFF (0)
colvars: - velocity_from_finite_differences   OFF (0)
colvars: - total_force   ON (3)
colvars: - total_force_calculation   ON (1)
colvars: - subtract_applied_force_from_total_force   ON (0)
colvars: - Jacobian_derivative   ON (1)
colvars: - hide_Jacobian_force   OFF (0)
colvars: - extended_Lagrangian   OFF (0)
colvars: - Langevin_dynamics   OFF (0)
colvars: - output_energy   OFF (0)
colvars: - output_value   ON (0)
colvars: - output_velocity   OFF (0)
colvars: - output_applied_force   ON (0)
colvars: - output_total_force   ON (0)
colvars: - lower_boundary   ON (1)
colvars: - upper_boundary   ON (1)
colvars: - hard_lower_boundary   OFF (0)
colvars: - hard_upper_boundary   OFF (0)
colvars: - reflecting_lower_boundary   OFF (0)
colvars: - reflecting_upper_boundary   OFF (0)
colvars: - grid   ON (0)
colvars: - running_average   OFF (0)
colvars: - correlation_function   OFF (0)
colvars: - scripted   OFF (0)
colvars: - custom_function   OFF (0)
colvars: - periodic   OFF (0)
colvars: - single_component   ON (0)
colvars: - scalar   ON (6)
colvars: - linear   ON (2)
colvars: - homogeneous   ON (0)
colvars: - multiple_timestep   OFF (0)
colvars:   * child 1
colvars:   Features of "cvc distance0001" ON/OFF (refcount)
colvars:   - active   ON (0)
colvars:   - scalar   OFF (0)
colvars:   - periodic   OFF (0)
colvars:   - defined_width   OFF (0)
colvars:   - defined_lower_boundary   ON (0)
colvars:   - defined_upper_boundary   OFF (0)
colvars:   - gradient   ON (2)
colvars:   - explicit_gradient   ON (0)
colvars:   - inverse_gradient   ON (2)
colvars:   - Jacobian_derivative   ON (1)
colvars:   - total_force_from_one_group   OFF (0)
colvars:   - debug_gradient   OFF (0)
colvars:   - use_minimum-image_with_PBCs   ON (0)
colvars:   - function_of_centers_of_mass   ON (1)
colvars:   - scalable_calculation   ON (0)
colvars:   - scalable_calculation_of_centers_of_mass   ON (1)
colvars:     * child 1
colvars:     Features of "atom group group1" ON/OFF (refcount)
colvars:     - active   OFF (0)
colvars:     - translational_fit   OFF (0)
colvars:     - rotational_fit   OFF (0)
colvars:     - fitting_group   OFF (0)
colvars:     - explicit_atom_gradient   ON (1)
colvars:     - fit_gradients   OFF (0)
colvars:     - atomic_forces   OFF (0)
colvars:     - scalable_group   ON (0)
colvars:     - scalable_group_center_of_mass   ON (1)
colvars:     * child 2
colvars:     Features of "atom group group2" ON/OFF (refcount)
colvars:     - active   OFF (0)
colvars:     - translational_fit   OFF (0)
colvars:     - rotational_fit   OFF (0)
colvars:     - fitting_group   OFF (0)
colvars:     - explicit_atom_gradient   ON (1)
colvars:     - fit_gradients   OFF (0)
colvars:     - atomic_forces   OFF (0)
colvars:     - scalable_group   ON (0)
colvars:     - scalable_group_center_of_mass   ON (1)
colvars: Features of "bias harmonic1" ON/OFF (refcount)
colvars: - active   ON (1)
colvars: - awake   ON (0)
colvars: - step_zero_data   OFF (0)
colvars: - apply_force   ON (0)
colvars: - bypass_extended_Lagrangian_coordinates   OFF (0)
colvars: - obtain_total_force   OFF (0)
colvars: - output_accumulated_work   OFF (0)
colvars: - history_dependent   OFF (0)
colvars: - time_dependent   OFF (0)
colvars: - require_scalar_variables   OFF (0)
colvars: - calculate_a_PMF   OFF (0)
colvars: - calculate_TI_samples   OFF (0)
colvars: - write_TI_samples_   OFF (0)
colvars: - write_TI_PMF   OFF (0)
colvars:   * child 1
colvars:   Features of "colvar d" ON/OFF (refcount)
colvars:   - active   ON (4)
colvars:   - awake   ON (0)
colvars:   - gradient   ON (3)
colvars:   - collect_gradient   OFF (0)
colvars:   - velocity_from_finite_differences   OFF (0)
colvars:   - total_force   ON (3)
colvars:   - total_force_calculation   ON (1)
colvars:   - subtract_applied_force_from_total_force   ON (0)
colvars:   - Jacobian_derivative   ON (1)
colvars:   - hide_Jacobian_force   OFF (0)
colvars:   - extended_Lagrangian   OFF (0)
colvars:   - Langevin_dynamics   OFF (0)
colvars:   - output_energy   OFF (0)
colvars:   - output_value   ON (0)
colvars:   - output_velocity   OFF (0)
colvars:   - output_applied_force   ON (0)
colvars:   - output_total_force   ON (0)
colvars:   - lower_boundary   ON (1)
colvars:   - upper_boundary   ON (1)
colvars:   - hard_lower_boundary   OFF (0)
colvars:   - hard_upper_boundary   OFF (0)
colvars:   - reflecting_lower_boundary   OFF (0)
colvars:   - reflecting_upper_boundary   OFF (0)
colvars:   - grid   ON (0)
colvars:   - running_average   OFF (0)
colvars:   - correlation_function   OFF (0)
colvars:   - scripted   OFF (0)
colvars:   - custom_function   OFF (0)
colvars:   - periodic   OFF (0)
colvars:   - single_component   ON (0)
colvars:   - scalar   ON (6)
colvars:   - linear   ON (2)
colvars:   - homogeneous   ON (0)
colvars:   - multiple_timestep   OFF (0)
colvars:     * child 1
colvars:     Features of "cvc distance0001" ON/OFF (refcount)
colvars:     - active   ON (0)
colvars:     - scalar   OFF (0)
colvars:     - periodic   OFF (0)
colvars:     - defined_width   OFF (0)
colvars:     - defined_lower_boundary   ON (0)
colvars:     - defined_upper_boundary   OFF (0)
colvars:     - gradient   ON (2)
colvars:     - explicit_gradient   ON (0)
colvars:     - inverse_gradient   ON (2)
colvars:     - Jacobian_derivative   ON (1)
colvars:     - total_force_from_one_group   OFF (0)
colvars:     - debug_gradient   OFF (0)
colvars:     - use_minimum-image_with_PBCs   ON (0)
colvars:     - function_of_centers_of_mass   ON (1)
colvars:     - scalable_calculation   ON (0)
colvars:     - scalable_calculation_of_centers_of_mass   ON (1)
colvars:       * child 1
colvars:       Features of "atom group group1" ON/OFF (refcount)
colvars:       - active   OFF (0)
colvars:       - translational_fit   OFF (0)
colvars:       - rotational_fit   OFF (0)
colvars:       - fitting_group   OFF (0)
colvars:       - explicit_atom_gradient   ON (1)
colvars:       - fit_gradients   OFF (0)
colvars:       - atomic_forces   OFF (0)
colvars:       - scalable_group   ON (0)
colvars:       - scalable_group_center_of_mass   ON (1)
colvars:       * child 2
colvars:       Features of "atom group group2" ON/OFF (refcount)
colvars:       - active   OFF (0)
colvars:       - translational_fit   OFF (0)
colvars:       - rotational_fit   OFF (0)
colvars:       - fitting_group   OFF (0)
colvars:       - explicit_atom_gradient   ON (1)
colvars:       - fit_gradients   OFF (0)
colvars:       - atomic_forces   OFF (0)
colvars:       - scalable_group   ON (0)
colvars:       - scalable_group_center_of_mass   ON (1)
colvars: Saving collective variables state to "test.colvars.state".
colvars: Features of "bias harm2" ON/OFF (refcount)
colvars: - active   ON (1)
colvars: - awake   ON (0)
colvars: - step_zero_data   OFF (0)
colvars: - apply_force   ON (0)
colvars: - bypass_extended_Lagrangian_coordinates   OFF (0)
colvars: - obtain_total_force   OFF (0)
colvars: - output_accumulated_work   OFF (0)
colvars: - history_dependent   OFF (0)
colvars: - time_dependent   OFF (0)
colvars: - require_scalar_variables   OFF (0)
colvars: - calculate_a_PMF   OFF (0)
colvars: - calculate_TI_samples   OFF (0)
colvars: - write_TI_samples_   OFF (0)
colvars: - write_TI_PMF   OFF (0)
colvars:   * child 1
colvars:   Features of "colvar d" ON/OFF (refcount)
colvars:   - active   ON (3)
colvars:   - awake   ON (0)
colvars:   - gradient   ON (2)
colvars:   - collect_gradient   OFF (0)
colvars:   - velocity_from_finite_differences   OFF (0)
colvars:   - total_force   ON (3)
colvars:   - total_force_calculation   ON (1)
colvars:   - subtract_applied_force_from_total_force   ON (0)
colvars:   - Jacobian_derivative   ON (1)
colvars:   - hide_Jacobian_force   OFF (0)
colvars:   - extended_Lagrangian   OFF (0)
colvars:   - Langevin_dynamics   OFF (0)
colvars:   - output_energy   OFF (0)
colvars:   - output_value   ON (0)
colvars:   - output_velocity   OFF (0)
colvars:   - output_applied_force   ON (0)
colvars:   - output_total_force   ON (0)
colvars:   - lower_boundary   ON (1)
colvars:   - upper_boundary   ON (1)
colvars:   - hard_lower_boundary   OFF (0)
colvars:   - hard_upper_boundary   OFF (0)
colvars:   - reflecting_lower_boundary   OFF (0)
colvars:   - reflecting_upper_boundary   OFF (0)
colvars:   - grid   ON (0)
colvars:   - running_average   OFF (0)
colvars:   - correlation_function   OFF (0)
colvars:   - scripted   OFF (0)
colvars:   - custom_function   OFF (0)
colvars:   - periodic   OFF (0)
colvars:   - single_component   ON (0)
colvars:   - scalar   ON (6)
colvars:   - linear   ON (2)
colvars:   - homogeneous   ON (0)
colvars:   - multiple_timestep   OFF (0)
colvars:     * child 1
colvars:     Features of "cvc distance0001" ON/OFF (refcount)
colvars:     - active   ON (0)
colvars:     - scalar   OFF (0)
colvars:     - periodic   OFF (0)
colvars:     - defined_width   OFF (0)
colvars:     - defined_lower_boundary   ON (0)
colvars:     - defined_upper_boundary   OFF (0)
colvars:     - gradient   ON (2)
colvars:     - explicit_gradient   ON (0)
colvars:     - inverse_gradient   ON (2)
colvars:     - Jacobian_derivative   ON (1)
colvars:     - total_force_from_one_group   OFF (0)
colvars:     - debug_gradient   OFF (0)
colvars:     - use_minimum-image_with_PBCs   ON (0)
colvars:     - function_of_centers_of_mass   ON (1)
colvars:     - scalable_calculation   ON (0)
colvars:     - scalable_calculation_of_centers_of_mass   ON (1)
colvars:       * child 1
colvars:       Features of "atom group group1" ON/OFF (refcount)
colvars:       - active   OFF (0)
colvars:       - translational_fit   OFF (0)
colvars:       - rotational_fit   OFF (0)
colvars:       - fitting_group   OFF (0)
colvars:       - explicit_atom_gradient   ON (1)
colvars:       - fit_gradients   OFF (0)
colvars:       - atomic_forces   OFF (0)
colvars:       - scalable_group   ON (0)
colvars:       - scalable_group_center_of_mass   ON (1)
colvars:       * child 2
colvars:       Features of "atom group group2" ON/OFF (refcount)
colvars:       - active   OFF (0)
colvars:       - translational_fit   OFF (0)
colvars:       - rotational_fit   OFF (0)
colvars:       - fitting_group   OFF (0)
colvars:       - explicit_atom_gradient   ON (1)
colvars:       - fit_gradients   OFF (0)
colvars:       - atomic_forces   OFF (0)
colvars:       - scalable_group   ON (0)
colvars:       - scalable_group_center_of_mass   ON (1)
colvars: The final output state file will be "test.colvars.state".
colvars: Saving collective variables state to "test.colvars.state".
colvars: The final output state file will be "test.colvars.state".
colvars: Saving collective variables state to "test.colvars.state".
colvars: ----------------------------------------------------------------------
colvars: Restarting from file "test.colvars.state".
colvars:   Restarting collective variable "d" from value: 11.9615
colvars:   Restarting abf bias "abf1" from step number 60.
colvars: ----------------------------------------------------------------------
colvars: Resetting the Collective Variables module.
colvars: ----------------------------------------------------------------------
colvars: Reading new configuration from file "test.in":
colvars: # units = "" [default]
colvars: # smp = on [default]
colvars: # colvarsTrajFrequency = 2
colvars: # colvarsRestartFrequency = 0 [default]
colvars: # scriptedColvarForces = off [default]
colvars: # scriptingAfterBiases = off [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = "d"
colvars:   Initializing a new "distance" component.
colvars:     # name = "" [default]
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # forceNoPBC = off [default]
colvars:     # scalable = on [default]
colvars:       Initializing atom group "group1".
colvars:       # name = "" [default]
colvars:       # centerReference = off [default]
colvars:       # rotateReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableForces = on [default]
colvars:       # enableFitGradients = on [default]
colvars:       Enabling scalable calculation for group "group1".
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "group1" defined with 1 atoms requested: total mass = 12.011, total charge = -0.1.
colvars:       Initializing atom group "group2".
colvars:       # name = "" [default]
colvars:       # centerReference = off [default]
colvars:       # rotateReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableForces = on [default]
colvars:       # enableFitGradients = on [default]
colvars:       Enabling scalable calculation for group "group2".
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "group2" defined with 1 atoms requested: total mass = 12.011, total charge = 0.07.
colvars:     # oneSiteSystemForce = off [default]
colvars:     # oneSiteTotalForce = off [default]
colvars:   All components initialized.
colvars:   # timeStepFactor = 1 [default]
colvars:   # width = 0.2
colvars:   # lowerBoundary = 12
colvars:   # upperBoundary = 32
colvars:   # hardLowerBoundary = off [default]
colvars:   # hardUpperBoundary = off [default]
colvars:   # expandBoundaries = off [default]
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputTotalForce = on
colvars:   # outputAppliedForce = on
colvars:   # subtractAppliedForce = on
colvars:   # runAve = off [default]
colvars:   # corrFunc = off [default]
colvars: ----------------------------------------------------------------------
colvars: Collective variables initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new "abf" instance.
colvars:   # name = "abf1" [default]
colvars:   # colvars = { d }
colvars:   # stepZeroData = off [default]
colvars:   # outputEnergy = off [default]
colvars:   # outputFreq = 60
colvars:   # timeStepFactor = 1 [default]
colvars:   # applyBias = on [default]
colvars:   # updateBias = on [default]
colvars:   # hideJacobian = off [default]
colvars:   Jacobian (geometric) forces will be included in reported free energy gradients.
colvars:   # fullSamples = 0
colvars:   # inputPrefix =  [default]
colvars:   # historyFreq = 0 [default]
colvars:   # shared = off [default]
colvars:   # maxForce =  [default]
colvars:   # integrate = on [default]
colvars:   Finished ABF setup.
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new "harmonic" instance.
colvars:   # name = "harmonic1" [default]
colvars:   # colvars = { d }
colvars:   # stepZeroData = off [default]
colvars:   # outputEnergy = on
colvars:   # outputFreq = 0 [default]
colvars:   # timeStepFactor = 1 [default]
colvars:   # writeTISamples = off [default]
colvars:   # writeTIPMF = off [default]
colvars:   # centers = { 12 }
colvars:   # targetCenters = { 12 } [default]
colvars:   # outputCenters = off [default]
colvars:   # forceConstant = 10
colvars:   # targetForceConstant = -1 [default]
colvars:   The force constant for colvar "d" will be rescaled to 250 according to the specified width (0.2).
colvars: ----------------------------------------------------------------------
colvars: Collective variables biases initialized, 2 in total.
colvars: ----------------------------------------------------------------------
colvars: Collective variables module (re)initialized.
colvars: ----------------------------------------------------------------------
colvars: Updating NAMD interface:
colvars: updating atomic data (0 atoms).
colvars: updating group data (2 scalable groups, 2 atoms in total).
colvars: Warning: before deleting colvar d, deleting related bias harmonic1
colvars: Warning: before deleting colvar d, deleting related bias abf1
